Storage/tests/Conformance/RetryConformanceTest.php

```diff
@@ -48,7 +48,7 @@ class RetryConformanceTest extends TestCase
     private static $cases = [];
 
     // Storage test bench URL. To be populated by an env variable.
-    private static $emaulatorUrl = 'https://storage.googleapis.com';
+    private static $emulatorUrl = '';
```

Comment:
Removing the default will cause the tests to fail unless emulator URL is specifically set. 
I suggest we have the default, and override if `STORAGE_EMULATOR_HOST` is provided.

Reply:
Good catch. 👍

Reply:
If you change the default emulator URL, you might also want to expose `$projectId` to the provided key file.

Reply:
So, I am fine in harcoding the `test` as project ID for now.  Since we are using InsecureCredentials and this specific project id.